FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c

```diff
@@ -3001,12 +2992,14 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t *pxSocket )
 				FreeRTOS_debug_printf( ( "prvTCPCreateStream: %cxStream created %lu bytes (total %lu)\n", xIsInputStream ? 'R' : 'T', uxLength, uxSize ) );
 			}
 
-			if( xIsInputStream != 0 )
+			if( xIsInputStream != pdFALSE )
 			{
+				iptraceMEM_STATS_CREATE( tcpRX_STREAM_BUFFER, pxBuffer, uxSize );
```

Comment:
Hi Hein,
I just have one question, does the memset on line 2986 `memset( pxBuffer, '\0', sizeof( *pxBuffer ) - sizeof( pxBuffer->ucArray ) );` affect the way the `iptraceMEM_STATS_CREATE( tcpRX_STREAM_BUFFER, pxBuffer, uxSize );` behaves?

Just making sure that moving `iptraceMEM_STATS_CREATE( tcpRX_STREAM_BUFFER, pxBuffer, uxSize );` from before `memset` to current location doesn't have any unintended effects.
Similarly for the `else` case...

Except that, all seems to be good. :)

Reply:
Good to notice this change in code, but no, the call to `memset()` and setting `pxBuffer->LENGTH` have no relation with `iptraceMEM_STATS_CREATE()`.
The latter will only log and register the pointer along with the type and the size.
Thanks.